bcftools.wdl

```diff
@@ -0,0 +1,22 @@
+version 1.0
+
+task Bcf2Vcf {
+    input {
+        File bcf
+        String outputPath
+    }
+    
+    command <<<
+        set -e
+        mkdir -p $(dirname ~{outputPath})
+        bcftools view ~{bcf} -O v -o ~{outputPath}
+    >>>
+    
+    output {
+        File OutputVcf = "~{outputPath}"
+    }
+    
+    runtime {
+        docker: "quay.io/biocontainers/bcftools:1.9--ha228f0b_3"
```

Comment:
Docker images tend to be made configurable in BioWDL.
Please, add an input like `String dockerImage = "quay.io/biocontainers/bcftools:1.9--ha228f0b_3" `
and change the following
```suggestion
        docker: dockerImage
```